explorer/client/src/pages/object-result/ObjectResult.tsx

```diff
@@ -1,31 +1,31 @@
 // Copyright (c) 2022, Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-import React, { useEffect, useState } from 'react';
+import { useEffect, useState } from 'react';
 import { useLocation, useParams } from 'react-router-dom';
+import { type GetObjectInfoResponse, isGetObjectInfoResponse } from 'sui.js';
 
 import ErrorResult from '../../components/error-result/ErrorResult';
 import theme from '../../styles/theme.module.css';
-import { DefaultRpcClient as rpc } from '../../utils/api/SuiRpcClient';
+import { DefaultRpcClient as rpc } from '../../utils/api/DefaultRpcClient';
+import { type Loadable } from '../../utils/loadState';
 import ObjectLoaded from './ObjectLoaded';
-import { type DataType } from './ObjectResultType';
 
-const DATATYPE_DEFAULT: DataType = {
-    id: '',
-    category: '',
-    owner: '',
-    version: '',
-    objType: '',
-    data: {
-        contents: {},
-        owner: { ObjectOwner: [] },
-        tx_digest: [],
-    },
+const DATATYPE_DEFAULT: Loadable<GetObjectInfoResponse> = {
     loadState: 'pending',
+    status: 'NotExists',
+    details: {
+        object: undefined,
+        objectRef: {
+            digest: '',
+            objectId: '',
+            version: 0,
+        },
+    },
 };
 
-function instanceOfDataType(object: any): object is DataType {
-    return object && ['id', 'version', 'objType'].every((x) => x in object);
+function instanceOfDataType(object: any): object is GetObjectInfoResponse {
```

Comment:
This is causing the version of the SuiExplorer website based on static `mock_data.json` to display errors for all objects. This is part of the reason why `yarn test` fails. The static objects will need to be updated to follow the schema implied by `isGetObjectInfoResponse`.